jbosstools/multiple/jbosstools-multiple.target

```diff
@@ -695,12 +700,6 @@
       <unit id="com.redhat.microprofile.jdt.quarkus.test" version="0.10.0"/>
```

Comment:
0.10.1 ?

Reply:
good catch, the 0.10.1 is not in the stable folder, I'll sync it anyway.